doc/README.md

```diff
@@ -83,7 +83,9 @@ The documentation about the scripts is [available here](scripts.md). More detail
 
 ### Pull-requests
 
-When Repairnator is configured to create a pull-request automatically, it uses the text of `TEXT_PR` from [AbstractRepairStep](https://github.com/eclipse/repairnator/blob/master/src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/AbstractRepairStep.java). In the future, it is planned to generate a specific piece of text to explain the patch (see [Explainable Software Bot Contributions: Case Study of Automated Bug Fixes](http://arxiv.org/pdf/1905.02597) ([doi:10.1109/BotSE.2019.00010](https://doi.org/10.1109/BotSE.2019.00010)))
```

Comment:
add a link to where this configuration is explained

Reply:
done!

Reply:
new repo: https://github.com/repairnator/r-hero-feedback-service